src/Features/Core/Portable/PullMemberUp/MembersPuller.cs

```diff
@@ -373,10 +373,8 @@ private static bool IsSelectedMemberDeclarationAlreadyInDestinationClass(ISymbol
             else
             {
                 var overrideMembersSet = new HashSet<ISymbol>();
-                for (var symbol = selectedMember; symbol != null; symbol = symbol.OverriddenMember())
-                {
+                for (var symbol = selectedMember; symbol != null; symbol = symbol.GetOverriddenMember())
```

Comment:
we had two methods that did the same thing.  i removed one and moved everyone on to the other.